rust/datafusion/src/execution/physical_plan/mod.rs

```diff
@@ -33,7 +33,8 @@ use arrow::{
 };
 use udf::ScalarFunction;
 
-/// Physical query planner
+/// Physical query planner that converts a `LogicalPlan` to an
+/// `ExecutionPlan` suitable for execution.
```

Comment:
Note that in the rustdoc format we can actually link to other items. e.g. `[ExecutionPlan](trait.ExecutionPlan.html)`